mavutil.py

```diff
@@ -228,7 +228,7 @@ def post_message(self, msg):
         msg._posted = True
         msg._timestamp = time.time()
         type = msg.get_type()
-        if type != 'HEARTBEAT' or (msg.type != mavlink.MAV_TYPE_GCS and msg.type != mavlink.MAV_TYPE_GIMBAL):
+        if type != 'HEARTBEAT' or (msg.type != mavlink.MAV_TYPE_GCS and msg.type != mavlink.MAV_TYPE_ONBOARD_CONTROLLER and msg.type != mavlink.MAV_TYPE_GIMBAL):
```

Comment:
also `type()` is a builtin function in Python so `msg_type` might be a better variable name